examples/04-Retrieval-Model.ipynb

```diff
@@ -0,0 +1,1847 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_path = '/workspace/data/train/*.parquet'
</pre><p>Can we have one base path and assume the other one are sub directories in it?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>